### PR TITLE
chore: update CoMapeo dependencies to final release versions

### PR DESCRIPTION
**Pull request by EvanHahn**
This updates CoMapeo dependencies to their final release versions.  Notably, several names changed. For example, `@mapeo/core` is now `@comapeo/core`.

There are no other changes in this repo other than renames and version bumps.

I did a quick manual test on a real Android phone and I was successfully able to make an observation.

### CODE REVIEW

File: src/frontend/hooks/useProjectInvitesListener.ts [hunk 1]
```diff
@@ -2,8 +2,11 @@ import {useCallback, useEffect, useState} from 'react';
 import {useApi} from '../contexts/ApiContext';
 import {useQueryClient} from '@tanstack/react-query';
 import {INVITE_KEY} from './server/invites';
-import {MapBuffers} from '@mapeo/core/dist/types';
-import {InviteInternal, InviteRemovalReason} from '@mapeo/core/dist/invite-api';
+import {MapBuffers} from '@comapeo/core/dist/types';
+import {
+  InviteInternal,
+  InviteRemovalReason,
+} from '@comapeo/core/dist/invite-api';
```
Comment (EvanHahn) [inline src/frontend/hooks/useProjectInvitesListener.ts:9]: Prettier broke this onto multiple lines.

File: src/frontend/sharedTypes/index.ts [hunk 2]
```diff
@@ -33,7 +33,7 @@ export type PhotoVariant = 'original' | 'thumbnail' | 'preview';
 export type CoordinateFormat = 'utm' | 'dd' | 'dms';
 export type MediaSyncSetting = 'previews' | 'everything';
 
-// Copied form /@mapeo/core/src/roles.js. Created an issue to eventuall expose this: https://github.com/digidem/mapeo-core-next/issues/532
+// Copied from @comapeo/core/src/roles.js. Created an issue to eventually expose this: https://github.com/digidem/mapeo-core-next/issues/532
```
Comment (EvanHahn) [inline src/frontend/sharedTypes/index.ts:36]: Couldn't resist fixing these comment typos.

**Comment by gmaclennan** [comment]
Just a correction about there being no changes: there is a change that moves Ajv to a dev dep for Mapeo schema. It should not change behaviour because ajv was being manually excluded from the metro bundle

**Comment by ErikSin** [comment]
@cimigree can you e2e this?

**Comment by EvanHahn** [comment]
> Just a correction about there being no changes: there is a change that moves Ajv to a dev dep for Mapeo schema. It should not change behaviour because ajv was being manually excluded from the metro bundle 

Good point. I tweaked the PR description to account for that.

**Review by cimigree — APPROVED** [review]
All current e2e tests passed in the same way as they did yesterday (no help text on details in observations)

**Comment by ErikSin** [comment]
im going to merge and cherry pick it onto release

**Pull request merged**